### DISCUSSION TITLE
customizable icons for groups

### DISCUSSION THREAD

**Discussion by ianmckenna87** [godotengine/godot-proposals#8470] [Editor]
Just a fun thought for a feature in case it's never been thought of, it would be cute to be able to create a custom icon for each group, so that the custom icon appears in the groups manager as well as next to the node in the tree instead of the default circle in a square.
![image](https://github.com/godotengine/godot-proposals/assets/132845829/e4ed118e-7349-4cb8-942b-02ec59a93598)
![image](https://github.com/godotengine/godot-proposals/assets/132845829/12e81e89-ba64-40e8-9fdb-4110a1c8d446)

**Comment by theraot** [comment]
Yet the node might be in multiple groups, what would it be in that case?

**Reply by ianmckenna87** [reply]
good point...

**Comment by theraot** [comment]
Since the icon must be configured somewhere, I'm guessing it would be in project settings.

See also:
- https://github.com/godotengine/godot-proposals/issues/907
- https://github.com/godotengine/godot-proposals/issues/3789
- https://github.com/godotengine/godot-proposals/issues/5846
